common/components/filter/template.njk

```diff
@@ -1,4 +1,4 @@
-<nav class="app-filter">
```

Comment:
Maybe add a test for this too. Should be some example you can just copy from other components.